Neos.Fusion/Resources/Private/Fusion/Root.fusion

```diff
@@ -167,6 +167,65 @@ prototype(Neos.Fusion:ResourceUri) {
   @exceptionHandler = 'Neos\\Fusion\\Core\\ExceptionHandlers\\AbsorbingHandler'
 }
 
+# Renders a link pointing to a controller/action
+#
+# Usage:
+# link = Neos.Fusion:Link.Action {
+#     package = 'Some.Package'
+#     controller = 'Standard'
+#     action = 'index'
+# }
+#
+prototype(Neos.Fusion:Link.Action) < prototype(Neos.Fusion:Component) {
+  package = ''
+  subpackage = ''
+  controller = ''
+  action = 'index'
```

Comment:
I set a default, because 'action' may not be empty... but maybe that is a bad idea?